cockatrice/src/pictureloader.cpp

```diff
@@ -420,20 +419,42 @@ bool PictureLoaderWorker::imageIsBlackListed(const QByteArray &picData)
     return md5Blacklist.contains(md5sum);
 }
 
+QNetworkReply *PictureLoaderWorker::makeRequest(const QUrl &url)
+{
+    QNetworkRequest req(url);
+    if (!picDownload) {
+        req.setAttribute(QNetworkRequest::CacheLoadControlAttribute, QNetworkRequest::AlwaysCache);
+    }
+    return networkManager->get(req);
+}
+
 void PictureLoaderWorker::picDownloadFinished(QNetworkReply *reply)
 {
     if (reply->error()) {
         qDebug() << "PictureLoader: [card: " << cardBeingDownloaded.getCard()->getName()
```

Comment:
having it say a download failed while it failed to get it from a cache makes no sense, I'd move this below the if statement there